x/blob/README.md

```diff
@@ -210,3 +194,16 @@ The steps in the
 function can be reverse engineered to submit blobs programmatically.
 
 <!-- markdownlint-enable MD010 -->
+
+## FAQ
+
+Q: Why do the responses from Comet BFT API endpoints fail to decode to valid transaction hashes?
+
+The response of CometBFT API endpoints (e.g. `/cosmos/base/tendermint/v1beta1/blocks/{block_number}`) will contain a field called `txs` with base64 encoded transactions. On most Cosmos based chains, one can base64 decode the transactions in the response and unmarshal the result into a `sdk.Tx` type. However, on Celestia, the `BlobTx` type is a wrapper around the Cosmos SDK transaction. For this reason, the transactions in the response can't directly be decoded and used. In order to use the response, one must unpack the transaction inside any `BlobTx`s. In other words:
+
+1. Base64 decode the transaction
+1. Check to see if the transaction is a `BlobTx` by unmarshalling it into a `BlobTx` type.
+   1. If it is a `BlobTx`, then unmarshal the `BlobTx`'s `Tx` field into a `sdk.Tx` type.
+   1. If it is not a `BlobTx`, then unmarshal the transaction into a `sdk.Tx` type.
+
+See this [Go Playground](https://go.dev/play/p/rdB1uVlQRml) for a code exmaple.
```

Comment:
I just realized that it's probably not best practice to link to a Go playground from our specs. I could replace the link with a new test in this repo if reviewers prefer that.

Reply:
https://github.com/celestiaorg/celestia-app/pull/2940/commits/78d249dfc2f7ade142e9e92cebd2487f61cbab47